streampark-flink/streampark-flink-kubernetes/pom.xml

```diff
@@ -96,6 +96,24 @@
             <version>${snakeyaml.version}</version>
         </dependency>
 
+        <dependency>
```

Comment:
I don't see where these jars are used in the new code, why do we need to add these jars ?